packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -56,6 +66,29 @@ function RecipientPicker(props: RecipientProps) {
   const [isKeyboardVisible, setKeyboardVisible] = useState(false)
   const [isSendToAddressWarningVisible, setSendToAddressWarningVisible] = useState(false)
 
+  const kitCall = useAsync(async () => {
+    return await getContractKitAsync(false)
+  }, [])
+
+  const resolveKitCall = useAsync(async () => {
```

Comment:
I think it's slightly nicer to read elsewhere if we do here instead: `const { result: resolveResult } = useAsync(`

This probably falls under personal preference though, so feel free to leave as-is if you like it more